Source/WebKit/UIProcess/ios/fullscreen/WKFullScreenWindowControllerIOS.mm

```diff
@@ -1562,6 +1586,14 @@ - (void)_configureSpatialFullScreenTransition
     });
 }
 
+- (void)_setOrnamentsHidden:(BOOL)hidden
+{
+    for (MRUIPlatterOrnament *ornament in [_parentWindowState ornamentProperties]) {
+        CGFloat originalAlpha = [[[_parentWindowState ornamentProperties] objectForKey:ornament] windowAlpha];
+        ornament.viewController.view.window.alpha = hidden ? 0.0 : originalAlpha;
```

Comment:
This could be rewritten to avoid map lookup when `hidden` is true, but not a very big deal.